README.md

```diff
@@ -83,7 +83,16 @@ From the repo:
       }'
    ```
 
-   If its the first time deploying the image then it should take few moments to download it.
+  If it's the first time deploying the image then it should take few moments to download it. You can also use the following variables to customize the sources of ovirt-flexvolume-driver and ovirt-volume-provisioner:
```

Comment:
Please change this line to something like: "To customize the images to deploy pass in those environment variables to the container invocation:"